### ISSUE TITLE
POC - Enter Statement of Work Data

### ISSUE THREAD

**Issue by MarsRomer** [bcgov/CONN-CCBC-portal#1369]
**Description:**
 As an analyst
I want to be able to enter the Statement of Work (SOW) data into the portal
So that it can be used for tracking the project, reporting, and announcing

This is a proof of concept ticket.
The signed funding agreement includes the SOW which which has a lot of data.  Once signed we need to track and report on some of the data in the SOW.  This ticket is to develop a solution for how to get the relevant and necessary data into the database.

# Acceptance Criteria:
- [ ] Some data from the SOW xls template is correctly imported into the database
 

![MicrosoftTeams-image (5).png](https://images.zenhubusercontent.com/63c8370399512153492c8e97/daf76659-5e59-4697-9058-36a4d28baffb)![MicrosoftTeams-image (6).png](https://images.zenhubusercontent.com/63c8370399512153492c8e97/32f49eb8-74a1-4af8-af07-8383a3322165)![MicrosoftTeams-image (7).png](https://images.zenhubusercontent.com/63c8370399512153492c8e97/e03e1a4e-ddff-4693-bf67-5fa3150f23db)

- [ ] .

**Page Behaviours**

- [ ] .
- [ ] .
- [ ] .

**Validations**

- [ ] .
- [ ] .
- [ ] .

**Actions**

- [ ] .
- [ ] .
- [ ] .

### Not included:
- .
- .

## Implementation Details
- **DB:**
  - .
  - .
- **Middleware:**
  - .
  - .
- **Frontend:**
  - .
  - .
- **Tests:**
  - .
  - .
- **Documentation:**
  - .
  - .

**Comment by elliottSlee** [comment]
Research done as of 3 April 2023
- [Required SoW data fields](https://miro.com/app/board/uXjVPtn4ugw=/?share_link_id=165436538674) are known
- These fields are the minimum to create the [desired reports](https://bcgov.sharepoint.com/:p:/r/teams/04708/Shared%20Documents/Research/CCBC%20Reports.pptx?d=w20a813d4ecb446ddada58620acb76033&csf=1&web=1&e=5xg7tY)
- Other fields such as Milestones are nice to have. They would allow project tracking and better analyst UX.
- [Low-fidelity wireframes are prepared](https://bcgov.sharepoint.com/:p:/r/teams/04708/Shared%20Documents/Research/CCBC%20Reports.pptx?d=w20a813d4ecb446ddada58620acb76033&csf=1&web=1&e=5xg7tY)
- Blocked by BC's decision of sticking to ISED's communities or adding BC's Geonames. ISED requires Communities. It is likely that BC required Geonames and # of Household/Geoname. If this is the case, we are blocked on the Excel template being prepared and tested by ISED.
- ISED ingests Statement of Work Excel file after negotiations to Access database. If there's a change between this ingestion & signing of Contribution Agreement, they re-ingest.
- Possible ways we could ingest the data:
  - Get access to their Access database
  - ISED exports from their Access database & send to BC
  - Ingest Excel file
  - Analyst enters information into form in CCBC portal
  - Recipient enters information into form in CCBC portal
